xml/security_ldap.xml

```diff
@@ -889,7 +663,258 @@ telephoneNumber: +49 1234 567-10</screen>
 Linux,ou=devel,dc=example,dc=com</screen>
   </sect2>
  </sect1>
- <sect1 xml:id="sec.security.ldap.info">
+
+<sect1 xml:id="sec.security.ldap.yast.usergr">
+  <title>Configuring LDAP Users and Groups in &yast;</title>
+<!--taroth 2019-02-25: let's see where to move/integrate the content from this
+    section in the end-->
+  <para>
+   The actual registration of user and group data differs only slightly from
```

Comment:
This block is a bit confusing to me? I don't really understand what it's trying to say.

Reply:
I think the sentence refers to the fact that you can administer a bunch of different users (local users, system users, NIS users, LDAP users, or custom users) with the same YaST dialog (you select which category to view and edit with 'Set Filter'). But as this information is not relevant for configuring LDAP users, I removed it. I also moved the information that groups can be administered in a similar way to the end of the procedure.